src/screens/ChatScreen/AnonymousChatScreen.tsx

```diff
@@ -104,7 +104,7 @@ const AnonymousChatScreen = () => {
       {selectedChannel ? (
         <ChatDetailHeader
           onAvatarPress={() => goToChatInfoScreen({from: ANONYMOUS})}
-          onBackPress={goBackFromChatScreen}
+          onBackPress={() => goBackFromChatScreen('ANONYMOUS')}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is passing a string parameter `'ANONYMOUS'` to the `goBackFromChatScreen` function. This seems to be a good approach as it allows for more flexibility in handling different types of chat screens. However, it would be better if you use an enum or constant for `'ANONYMOUS'` instead of a hardcoded string. This will help prevent potential bugs due to typos and make the code easier to maintain.
```diff
- onBackPress={() => goBackFromChatScreen('ANONYMOUS')}
+ onBackPress={() => goBackFromChatScreen(ChatTypes.ANONYMOUS)}
```
In this case, `ChatTypes` could be an enum or object that holds all your chat type constants.


<!-- This is an auto-generated comment by OSS CodeRabbit -->